packit/local_project.py

```diff
@@ -458,20 +458,27 @@ def _fetch_as_branch(
         """
         remote = self.remote or "origin"
         self.git_repo.remotes[remote].fetch(f"{remote_ref}:{local_ref}")
-        self.git_repo.create_head(local_branch, f"{remote}/{local_branch}")
+        # overwrite the local checkout when needed, remote is always accurate
+        self.git_repo.create_head(local_branch, f"{remote}/{local_branch}", force=True)
 
-    def checkout_pr(self, pr_id: Union[str, int]):
+    def checkout_pr(self, pr_id: Union[str, int]) -> git.Commit:
         """
-        Fetch selected PR and check it out.
+        Fetch selected PR and check it out in a local branch pr/{pr_id}
+
+        Args:
+            pr_id: ID of the PR we are merging.
+
+        Returns:
+            git.Commit HEAD of the checked out PR
```

Comment:
We have type annotations, you don't need to put type in the comments.

Reply:
also missing periods at the end, here and in the comment below for `pr_head` argument

Reply:
roger, will update